torchbenchmark/util/distributed/README.md

```diff
@@ -0,0 +1,60 @@
+## Distributed Training Paradigms
+
+This folder contains native distributed training paradigms from PyTorch. The
+current structure makes distributed training paradigms orthogonal to models.
+Users can specify model and trainer when launching experiments.
+
+
+```
+$ python -m torchbenchmark.util.distributed.submit -h
+usage: submit.py [--ngpus NGPUS] [--nodes NODES] [--timeout TIMEOUT] [--profiler PROFILER] [--partition PARTITION] [--job_dir JOB_DIR] [--model MODEL]
+                 [--trainer TRAINER]
+
+Submitit for PyTorch Distributed Benchmark
+
+optional arguments:
+  --ngpus NGPUS         Number of gpus to request on each node
+  --nodes NODES         Number of nodes to request
+  --timeout TIMEOUT     Duration of the job
+  --profiler PROFILER   Measure with PyTorch Profiler. Disabled by default, as it crashes on AWS
+  --partition PARTITION
+                        The Slurm partition to submit to
```

Comment:
Just curious, what is a partition in Slurm? Why it is set to `train` in the command example below?

Reply:
a partition is a pool of resources (nodes) in Slurm. In our cluster, we have a pool for dev and a pool for train.

Reply:
If we are only supporting single machine multiple GPU this half, does that mean slurm is not needed? (I tried on single machine multiple GPU without slurm (only installed submitit), and it seems working well)

Reply:
I am not sure about that. I haven't used submitit without Slurm before.